README.md

```diff
@@ -1,6 +1,6 @@
-# Build a script that lets agent schedule a callback during a call
+# Build a script that lets agents schedule a callback during a call
 
-> View the full [Build a script that lets agent schedule a callback during a call Blueprint](https://developer.mypurecloud.com/blueprints/schedule-callback-via-script/ "Goes to the Build a script that lets agent schedule a callback during a call Blueprint") in the Genesys Cloud Developer Center.
+> View the full [Build a script that allows agents to schedule a callback during a call Blueprint](https://developer.mypurecloud.com/blueprints/schedule-callback-via-script/ "Goes to the Build a script that allows agents to schedule a callback during a call Blueprint") in the Genesys Cloud Developer Center.
 
 This Genesys Cloud Developer Blueprint provides instructions for building a script to let agents schedule a callback via Script while a voice interaction is ongoing. You can deploy all the components used in this solution with the Terraform Genesys Cloud CX as Code provider.
```

Comment:
Line 5: 
This Genesys Cloud Developer Blueprint builds a script that allows agents to schedule a callback during a voice interaction. You deploy all the components used in this solution with the Terraform Genesys Cloud CX as Code provider.

Reply:
"voice interaction"... or "call" I see both variations :-)